app/v2/support/DownstreamResponseMappingSupport.scala

```diff
@@ -27,9 +27,15 @@ trait DownstreamResponseMappingSupport {
   final def mapDownstreamErrors[D](errorCodeMap: PartialFunction[String, MtdError])(downstreamResponseWrapper: ResponseWrapper[DownstreamError])(
       implicit logContext: EndpointLogContext): ErrorWrapper = {
 
-    lazy val defaultErrorCodeMapping: String => MtdError = { code =>
-      logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - No mapping found for error code $code")
-      InternalError
+    lazy val defaultErrorCodeMapping: String => MtdError = {
+      case "UNMATCHED_STUB_ERROR" =>{
+        logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - No matching stub was found")
+        RuleIncorrectGovTestScenarioError
+      }
+      case code => {
```

Comment:
Could probably do with a test in `DownstreamResponseMappingSupportSpec` to cover this.

Reply:
I believe that's already covered [here](https://github.com/hmrc/property-business-api/pull/361/files#diff-57330e552c06302dea0188f3c119e626d951939bf8201eac35ac94a3afd240a3R62)?

Reply:
So it is! Cool.